Dockerfile

```
@@ -0,0 +1,20 @@
+# Stage1
+FROM node:lts as node
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm ci
```

Should probably be -

```suggestion
RUN npm ci --production
```

---

I'm afraid I'm no node expert, just dockerizing this by using the install instructions.

Adding the `--production` flag leads to a error in the `RUN ng build` section: 
```
An unhandled exception occurred: Cannot find module '@angular-devkit/build-angular/package.json'
Require stack:
- /usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/architect/node/node-modules-architect-host.js
- /usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/architect/node/index.js
- /usr/local/lib/node_modules/@angular/cli/models/architect-command.js
- /usr/local/lib/node_modules/@angular/cli/commands/build-impl.js
- /usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/schematics/tools/export-ref.js
- /usr/local/lib/node_modules/@angular/cli/node_modules/@angular-devkit/schematics/tools/index.js
- /usr/local/lib/node_modules/@angular/cli/utilities/json-schema.js
- /usr/local/lib/node_modules/@angular/cli/models/command-runner.js
- /usr/local/lib/node_modules/@angular/cli/lib/cli/index.js
- /usr/local/lib/node_modules/@angular/cli/lib/init.js
- /usr/local/lib/node_modules/@angular/cli/bin/ng
See "/tmp/ng-eIwsIt/angular-errors.log" for further details.
```
Any idea?

---

Doh, my bad - forget that comment, this is the "build" stage - I totally missed it as used to CAPITAL_LETTER_NAMES for it lol

---

Yeah, I try to make this image as tiny as possible as it's essentially a static site :)